apps/system-e2e/src/tests/islandis/service-portal/smoke/access-control-consents.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { BrowserContext, expect, test } from '@playwright/test'
+
+import { urls } from '../../../../support/urls'
+import { session } from '../../../../support/session'
+
+const homeUrl = `${urls.islandisBaseUrl}/minarsidur/adgangsstyring/gagnaoflun`
+test.use({ baseURL: urls.islandisBaseUrl })
+
+test.describe('Service portal access control concents', () => {
+  let contextGranter: BrowserContext
+
+  test.beforeAll(async ({ browser }) => {
+    contextGranter = await session({
+      browser,
+      storageState: 'service-portal-faereyjar.json',
+      homeUrl,
+      phoneNumber: '0102399',
+    })
+  })
+
+  test.afterAll(async () => {
+    await contextGranter.close()
+  })
+
+  test('can manage consents ', async () => {
+    // Arrange
+    const page = await contextGranter.newPage()
+    await page.goto(homeUrl)
+    const actionCardDataTestId = 'consent-accordion-card'
+
+    await test.step('See consents overview and content', async () => {
+      await expect(
+        page.getByRole('heading', { name: 'Gagnaöflun' }),
+      ).toBeVisible()
+
+      // Arrange - wait for the consent list to load
+      await page.waitForResponse(
+        (resp) =>
+          resp.url().includes('/api/graphql?op=GetConsentList') &&
+          resp.status() === 200,
+      )
```

Comment:
Might want to move this above the heading check to avoid potential race conditions.